demo/src/main/res/drawable/bg_country_selection_modal.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <solid android:color="?kevinSecondaryBackgroundColor"/>
+    <corners android:topLeftRadius="16dp"
+        android:topRightRadius="16dp"/>
```

Comment:
format xmls also, because I see, that there should be space before `/>`. Use **CMD+OPTION+L**